docs/en/latest/how-to-build.md

```diff
@@ -21,351 +25,220 @@ title: Installing Apache APISIX
+## Next steps
 
-You can modify the Admin API key to secure your Apache APISIX deployment.
+### Updating Admin API key
 
-This can be done by modifying the `apisix.admin_key` in `conf/config.yaml` and restarting the service.
+It is recommended to modify the Admin API key to ensure security.
 
-```yaml
+You can update your configuration file as shown below:
+
+```yaml title="conf/config.yaml"
 apisix:
-  # ... ...
   admin_key
     -
       name: "admin"
-      key: abcdefghabcdefgh # Modify the original key to abcdefghabcdefgh
+      key: newsupersecurekey
       role: admin
 ```
 
-Then to access the Admin API, you can use the above key.
-
-```shell
-curl http://127.0.0.1:9080/apisix/admin/routes?api_key=abcdefghabcdefgh -i
-```
-
-A status code of 200 in the returned result will indicate that the access was successful.
+Now, to access the Admin API, you can use the new key:
 
 ```shell
-HTTP/1.1 200 OK
-Date: Fri, 28 Feb 2020 07:48:04 GMT
-Content-Type: text/plain
-... ...
-{"node":{...},"action":"get"}
+curl http://127.0.0.1:9080/apisix/admin/routes?api_key=newsupersecurekey -i
 ```
 
-If the key you entered does not match the value of `apisix.admin_key` in `conf/config.yaml`, a response with a status code 401 will indicate that the access failed.
+### Adding APISIX systemd unit file
 
-```shell
-curl http://127.0.0.1:9080/apisix/admin/routes?api_key=wrong-key -i
-```
-
-```shell
-HTTP/1.1 401 Unauthorized
-Date: Fri, 28 Feb 2020 08:17:58 GMT
-Content-Type: text/html
-... ...
-{"node":{...},"action":"get"}
-```
-
-## Step 6: Build OpenResty for Apache APISIX
-
-Some features require additional NGINX modules to be introduced into OpenResty.
-
-If you need these features, you can build APISIX OpenResty. You can refer to the source of [api7/apisix-build-tools](https://github.com/api7/apisix-build-tools) for setting up your build environment and building APISIX OpenResty.
-
-## Step 7: Add Systemd unit file for Apache APISIX
-
-If you are using CentOS and installed [Apache APISIX via the RPM package](#installation-via-rpm-repository-centos), the configuration file will already be in place and you can run the following command directly.
+If you installed APISIX via RPM, the APISIX unit file will already be configured and you can start APISIX by:
 
 ```shell
 systemctl start apisix
 systemctl stop apisix
 ```
 
-If you installed Apache APISIX by other methods, please refer to the [configuration file template](https://github.com/api7/apisix-build-tools/blob/master/usr/lib/systemd/system/apisix.service) for a modification guide and copy it to the `/usr/lib/systemd/system/apisix.service` path.
+If you installed APISIX through other methods, you can create `/usr/lib/systemd/system/apisix.service` and add the [configuration from the template](https://github.com/api7/apisix-build-tools/blob/master/usr/lib/systemd/system/apisix.service).
+
+See the [Getting Started](./getting-started.md) guide for a quick walk-through of using APISIX.
```

Comment:
```suggestion
See the [Getting Started](./getting-started.md#Create-a-Route) guide for a quick walk-through of using APISIX.
```

Reply:
We can just link to the whole doc right? Even if someone installs APISIX, they might not be knowing some of the basics.

Reply:
ok, you're right.